src/VirtoCommerce.CustomerExportImportModule.Web/Scripts/blades/import-preview.js

```diff
@@ -98,19 +98,27 @@ angular.module('virtoCommerce.customerExportImportModule')
                         column.cellTooltip = getCellTooltip;
                         column.headerCellClass = 'br-0 font-weight-500 fs-13';
                     });
-                    const fullNameColumn = _.findWhere(gridApi.grid.options.columnDefs, {name: 'contactFullName'});
-                    const idColumn = _.findWhere(gridApi.grid.options.columnDefs, {name: 'contactId'});
-                    const birthdayColumn = _.findWhere(gridApi.grid.options.columnDefs, { name: "birthday" });
-                    fullNameColumn.pinnedLeft = true;
-                    fullNameColumn.cellClass = "pl-7 bl-0 font-weight-500 fs-12";
-                    fullNameColumn.headerCellClass = "pl-7 font-weight-500 fs-13";
-                    if (idColumn) {
-                        idColumn.enablePinning = true;
-                        idColumn.hidePinLeft = false;
+
+                    if (blade.dataType.value === 'Contact') {
+                        $scope.fullNameColumn = _.findWhere(gridApi.grid.options.columnDefs, {name: 'contactFullName'});
+                        $scope.idColumn = _.findWhere(gridApi.grid.options.columnDefs, {name: 'contactId'});
+                        const birthdayColumn = _.findWhere(gridApi.grid.options.columnDefs, { name: "birthday" });
+                        if (birthdayColumn) {
+                            birthdayColumn.cellTemplate = "birthday.col.html";
+                        }
+                    } else {
+                        $scope.fullNameColumn = _.findWhere(gridApi.grid.options.columnDefs, {name: 'organizationName'});
+                        $scope.idColumn = _.findWhere(gridApi.grid.options.columnDefs, {name: 'organizationId'});
                     }
-                    if (birthdayColumn) {
-                        birthdayColumn.cellTemplate = "birthday.col.html";
+
+                    $scope.fullNameColumn.pinnedLeft = true;
+                    $scope.fullNameColumn.cellClass = "pl-7 bl-0 font-weight-500 fs-12";
+                    $scope.fullNameColumn.headerCellClass = "pl-7 font-weight-500 fs-13";
```

Comment:
May we rename fullNameColumn to something like pinnedColum? It confusing what organization name is treated as "full name"

Reply:
done